api/python/quilt3/data_transfer.py

```diff
@@ -473,7 +474,7 @@ def run_task(idx, func, *args):
                 idx_to_futures[idx].append(future)
 
         def worker(idx, src, dest, size):
-            if stopped:
```

Comment:
Why change the name of this variable?

Reply:
shadows the outer variable `stopped` since I moved the operation inside an internal function `copy_internal` with `stopped` as a parameter.